crates/sui-types/src/object.rs

```diff
@@ -554,8 +554,14 @@ impl Display for Owner {
             Self::Immutable => {
                 write!(f, "Immutable")
             }
-            Self::Shared { .. } => {
-                write!(f, "Shared")
+            Self::Shared {
+                initial_shared_version,
+            } => {
+                write!(
+                    f,
+                    "Shared(initial_shared_version: {})",
```

Comment:
this is used in sui-tool, where I think I'd prefer the more concise option of `Shared(N)`, if its all the same to you

Reply:
All the same to me -- will update.

Reply:
Are we okay with the non-hex value here? We definitely want it for tests, I'm just not sure why hex was used in the display normally